app/models/application_qualification.rb

```diff
@@ -60,6 +60,13 @@ class ApplicationQualification < ApplicationRecord
   validates :qualification_type, length: { maximum: MAX_QUALIFICATION_TYPE_LENGTH }, allow_blank: true
   validates :non_uk_qualification_type, length: { maximum: MAX_QUALIFICATION_TYPE_LENGTH }, allow_blank: true
 
+  enum enic_reason: {
```

Comment:
I don't think we need these comments in the code. I'd recommend making sure an explanation is available in the commit message and the PR description